src/utils/getCategoriesOption.js

```diff
@@ -1,8 +1,11 @@
 const getCategoriesOptions = (categories) => {
-  if (categories) {
+  if (Array.isArray(categories)) {
     return categories.map((item) => (item === 'null' ? null : item))
-  } else {
-    return
+  }
+
+  if (typeof categories === 'string') {
+    return categories === 'null' ? [null] : [categories]
```

Comment:
categories cannot bu null here, because typeof null === 'string' is false
typeof null equals to 'object'